cluster/nats_rpc_client.go

```diff
@@ -197,6 +212,7 @@ func (ns *NatsRPCClient) Call(
 		}
 		return nil, err
 	}
+	logger.Log.Debug("returning response from remote (client)")
```

Comment:
Why do we need this log line, even being a debug it seems to be useless

Reply:
we don't, this was accidentally left during testing, thanks for the catch, i'll remove it